### ISSUE TITLE
Uncaught (in promise) TypeError: serverConnection is undefined

### ISSUE THREAD

**Issue by jjsarton** [jech/galene#237]
On galene.org:8443/groups/public:

1) Launch the page

2) open setting and set filter "Background blur"

Firefox show:

```
Uncaught (in promise) TypeError: serverConnection is undefined
    findUpMedia https://galene.org:8443/galene.js:1764
    <anonymous> https://galene.org:8443/galene.js:637
[galene.js:1764:19](https://galene.org:8443/galene.js)
    findUpMedia https://galene.org:8443/galene.js:1764
    <anonym> https://galene.org:8443/galene.js:637
    receiveMessage resource://gre/actors/SelectChild.sys.mjs:260
    receiveMessage resource://gre/actors/SelectChild.sys.mjs:498
```

**Comment by jech** [comment]
Good catch, thanks.

To clarify, this happens if you pick a filter *before* you login.  The easy solution would be to disable the filter menu before login.  I'll see if I can find a better solution

**Comment by jjsarton** [comment]
My intention was to explicitly set the background filter before login. The easy solution is not a good solution.

**Comment by jech** [comment]
Agreed.

**Comment by jech** [comment]
Fixed in 01931b27e66fd400e7990b7e4c8aa56579e10176, thanks for the report.
